app/src/main/java/practicum/getfitla_v3/MainActivity.java

```diff
@@ -1,13 +1,10 @@
 package practicum.getfitla_v3;
 
 import android.content.Intent;
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
-import android.support.v7.widget.RecyclerView;
+import android.support.v7.app.AppCompatActivity;
```

Comment:
Ditto. Why does this remove RecyclerView? Or is it an unused import?

Reply:
Yes, it's unused.